sensors/gc0308.c

```diff
@@ -184,6 +184,8 @@ static int set_framesize(sensor_t *sensor, framesize_t framesize)
     uint16_t h = resolution[framesize].height;
     uint16_t row_s = (resolution[FRAMESIZE_VGA].height - h) / 2;
     uint16_t col_s = (resolution[FRAMESIZE_VGA].width - w) / 2;
+    (void)row_s;
+    (void)col_s;
```

Comment:
In which case do you get a warning here? Regardless if `CONFIG_GC_SENSOR_SUBSAMPLE_MODE` is set, both variables are used. Am I missing something?

Reply:
Ah sorry, I see it :)